packages/orchestration/src/examples/auto-stake-it-tap-kit.js

```diff
@@ -0,0 +1,155 @@
+import { M, mustMatch } from '@endo/patterns';
+import { E } from '@endo/far';
+import { VowShape } from '@agoric/vow';
+import { makeTracer } from '@agoric/internal';
+import { atob } from '@endo/base64';
+import { ChainAddressShape } from '../typeGuards.js';
+
+const trace = makeTracer('AutoStakeItTap');
+
+/**
+ * @import {IBCChannelID, VTransferIBCEvent} from '@agoric/vats';
+ * @import {VowTools} from '@agoric/vow';
+ * @import {Zone} from '@agoric/zone';
+ * @import {TargetApp} from '@agoric/vats/src/bridge-target.js';
+ * @import {ChainAddress, CosmosValidatorAddress, Denom, OrchestrationAccount, StakingAccountActions} from '@agoric/orchestration';
+ * @import {FungibleTokenPacketData} from '@agoric/cosmic-proto/ibc/applications/transfer/v2/packet.js';
+ */
+
+/**
+ * @typedef {{
+ *   stakingAccount: ERef<OrchestrationAccount<any> & StakingAccountActions>;
+ *   localAccount: ERef<OrchestrationAccount<{ chainId: 'agoric' }>>;
+ *   validator: CosmosValidatorAddress;
+ *   localChainAddress: ChainAddress;
+ *   remoteChainAddress: ChainAddress;
+ *   sourceChannel: IBCChannelID;
+ *   remoteDenom: Denom;
+ *   localDenom: Denom;
+ * }} StakingTapState
+ */
+
+const StakingTapStateShape = {
+  stakingAccount: M.remotable('CosmosOrchestrationAccount'),
+  localAccount: M.remotable('LocalOrchestrationAccount'),
+  validator: ChainAddressShape,
+  localChainAddress: ChainAddressShape,
+  remoteChainAddress: ChainAddressShape,
+  sourceChannel: M.string(),
+  remoteDenom: M.string(),
+  localDenom: M.string(),
+};
+harden(StakingTapStateShape);
+
+/**
+ * @param {Zone} zone
+ * @param {VowTools} vowTools
+ */
+const prepareMakeStakingTapKit = (zone, { watch }) => {
+  return zone.exoClassKit(
+    'StakingTap',
```

Comment:
not sure how much this matters, but it would be good to be consistent
```suggestion
    'StakingTapKit',
```